src/style.css

```diff
@@ -1 +1,55 @@
-/* add styles here */
+/* #region head */
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region starts */
+
+/* .all__stars {
+  display: flex;
+  height: 100vh;
+  width: 100vh;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} */
+
+.stars {
```

Comment:
Remember to be consistent with your margins to avoid potential margin collapse. You have a margin-right on `.stars__star` elements, but you should decide if you want to use margin-right or margin-left consistently throughout your CSS.